test/shared/src/main/scala/zio/test/laws/package.scala

```diff
@@ -102,11 +102,11 @@ package object laws {
 
   object Laws {
     type Law1[-Caps[_]]  = ZLaws.Law1[Caps]
-    type Law1M[-Caps[_]] = ZLaws.Law1M[Caps, Any]
+    type Law1IO[-Caps[_]] = ZLaws.Law1ZIO[Caps, Any]
```

Comment:
I think I would leave this as `ZIO` since `Laws.Law1` as opposed to `ZLaws.Law1` indicates it uses the environment and `ZIO` versus not `ZIO` indicates it uses a ZIO workflow.

Reply:
Ooops, missed the `Z` when renaming it...

Reply:
I have doubled check and this should be the only place I misspelled `ZIO`